docs/fleet/provision.md

```diff
@@ -1,278 +1,186 @@
-   - You can use an existing configuration file from a deployed machine, or adapt such a file as needed to fit the specifications of your new machine.
-
-1. Place this file in the following location on the machine you wish to install the Viam Agent to: <file>/etc/viam.json</file>
-
-1. Run the following command to install the Viam Agent on your machine:
-
-   ```sh {class="command-line" data-prompt="$"}
-   sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/install.sh)"
-   ```
+Using `agent-provisioning`, this company can ship their machines directly to customers with `viam-agent` installed.
+When a customer sets up their machine, the provisioning subsystem installs `viam-server`.
+By having the end customer set up the machine, the company:
 
-{{% /tab %}}
-{{< /tabs >}}
+- eliminates factory setup time
+- allows for tailored configurations per customer as needed
 
-### Update the Viam Agent and `viam-server`
+{{< alert title="Support Notice" color="note" >}}
 
-The Viam Agent automatically updates both itself and `viam-server` as new updates are released.
-You can also configure update behavior for the Agent and `viam-server` using the [Viam app](https://app.viam.com/).
+Provisioning works on Bullseye, Debian Bookworm or newer.
+For Bullseye, the installation of `viam-agent` changes the network configuration to use NetworkManager.
 
-{{< alert title="Important" color="note" >}}
-When the Viam Agent updates either itself or `viam-server`, you must restart these services in order to use the new version.
-When you stop or restart the Viam Agent, the agent will stop or restart `viam-server` as well.
 {{< /alert >}}
 
-### Manage the Viam Agent
+## How an end user would use it
 
-The Viam Agent is installed as a `systemd` service named `viam-agent`.
+End users that receive a machine, use either a captive portal through a browser or a web or mobile app to complete the machine setup.
 
-- To start the Viam Agent:
+One option is to use the [Viam mobile app](/fleet/control/#control-interface-in-the-viam-mobile-app).
+The Viam mobile app allows end users to create a new machine in the app, and `agent-provisioning` will then install `viam-server` and run it with a provided configuration.
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl start viam-agent
-  ```
+To add your branding, you can build your own web or mobile app and use one of Viam's SDKs to connect to `viam-agent` and provision your machines:
 
-- To stop the Viam Agent:
+- For mobile applications (existing or new) use the [Flutter SDK](https://flutter.viam.dev/viam_protos.provisioning.provisioning/ProvisioningServiceClient-class.html)
+- For web applications, use the [TypeScript SDK](https://github.com/viamrobotics/viam-typescript-sdk/blob/main/src/app/provisioning-client.ts).
```

Comment:
I don't know. I've been told the TypeScript SDK supports provisioning. But while reading your above comment I was also wondering if that would then be the captive portal it would replace... @ale7714 do you have more info on this?